pyvista/utilities/geometric_objects.py

```diff
@@ -1058,9 +1071,7 @@ def Rectangle(points=[[1, 0, 0], [1, 1, 0], [0, 1, 0], [0, 0, 0]]):
     check_valid_vector(points[3], 'points[3]')
 
     cells = np.array([[4, 0, 1, 2, 3]], ctypes.c_long)
```

Comment:
Made a new issue in #1605.

We should change it in a separate PR.  While I doubt that this will cause any issues, if it's a separate PR then we can `git bisect` in the rare case that it turns out to have been an issue.